docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,498 @@
+        this.evt.on('lcSuspend', this.returnToMain);
+        ...
+    }
+    deactivate() {
+        ...
+        this.evt.off('lcSuspend', this.returnToMain);
+        ...
+    }
+    myPrelaunch(winId, orig) {
+        this.org = orig;
+        ...
+    }
+    resume(winId) {
+        ...
+    }
+    returnToMain(winId) {
+        addonAPI.window.loadURL(winId, this.org);
+    }
+    ```
+
+To test the implementation, follow these steps:
+
+1. In the Web application, press the **Home** key to enter the suspend state.
+2. Long-press the **Home** key and select **Basic UI** to enter the resume state.
+
+### Return to main page of Web application using back key
+
+When you press the **Back** key in the `hello.html` page, nothing happens. It is usually expected to return to a previous page from the current page after pressing the **Back** key.
+To implement it, modify `hello.html` as shown in the following code:
+
+```
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+    <script type="text/javascript">
+        window.onload = function() {
+            document.addEventListener("tizenhwkey", function(e) {
+                if (e.keyName == "back") try {
+                    // your action
+                } catch (t) {}
+            });
+        }
+    </script>
+</head>
+```
+
+You can add your code in the `try` block to enable the **Back** key functionality in the `hello.html` page. In this case, it must return to the main page of the Web application. The preceding code is working on the renderer process. However, the API loadURL that you used earlier to change the page of the current window is working at the browser process. The browser process must know when the **Back** key is pressed.
+
+There is a set of APIs for the messaging system between the two processes in the Addon APIs. You must specify the name of the message as you want. In this example, you can use `backkey` as the name. Then, in `hello.html`, it must send a message `backkey` to the browser process when the **Back** key is pressed. You can replace your action using the following code:
+
+```
+require('addonapi').messaging.send('backkey');
+```
+
+Additionally, you need to add some code to handle the message in `myPrelaunch()` of `tmplDefault.js` at browser process:
+
+```
+myPrelaunch(winId, orig) {
+    ...
+    var this_ = this;
+    addonAPI.messaging.on('backkey', (event, msg) => {
+        addonAPI.window.loadURL(winId, this_.org);
+    });
+}
+```
+
+When you test with newly built package including the code above, `back key` in `hello.html` will bring you to the main page of the Web application. I believe you remember that you added code to return to the main page when the suspend event occurs. Now, you can remove the method `returnToMain()` and all other code related to `lcSuspend` because only handling `back key` in `hello.html` is sufficient. Following is the final code:
```

Comment:
@hyundukkim 
Are we trying to say:
Now, when you test the newly built package after modifying hello.html, `back key` in `hello.html` will take you to the main page of the Web application.
If you are implementing `back key` in `hello.html`, you need to remove the `returnToMain()` method and all other code related to `lcSuspend`, which was added while resolving the issues of addon events.
The final implementation is as shown in the following code:

Please verify and share your feedback so that we can rephrase it.

Reply:
Your understanding is right.

Reply:
Thank you

Reply:
@hyundukkim,
Can we rephrase as:
Now, when you test the newly built package after modifying the **hello.html** file, `back key` in **hello.html** will take you to the main page of the Web application.
If you are implementing `back key` in **hello.html**, you need to remove the `returnToMain()` method and all other code related to `lcSuspend`, which was added while resolving the issues of the addon events.
The final implementation is as shown in the following code:

Please verify.

**Global Comment**:
It is recommended to bold the file names instead of using code tags for the file names.
Please change `hello.html` to **hello.html** throughout the file.

Reply:
@Reni-90 ,

rephrasing: **done**
changing file names as bold: **done** for hello.html, tmplDefault.js, and config.xml

**Question:** 
How about the file path like 'shared/res' ? Do I need to change them as bold, too ?

Reply:
@hyundukkim
You don't need to apply the bold font to the file path. So we can keep 'shared/res'.

Reply:
Okay. Thanks for clarification.